lte/gateway/c/oai/lib/hashtable/hashtable_uint64.c

```diff
@@ -433,7 +433,7 @@ hashtable_key_array_t* hashtable_uint64_ts_get_keys(
     return NULL;
   }
   ka       = calloc(1, sizeof(hashtable_key_array_t));
-  ka->keys = calloc(hashtblP->num_elements, sizeof(hash_key_t*));
+  ka->keys = calloc(hashtblP->num_elements, sizeof(hash_key_t));
 
   while ((ka->num_keys < hashtblP->num_elements) && (i < hashtblP->size)) {
```

Comment:
Same comment form #5327
uninitialized access to  ka->num_keys and ea->num_elements below